src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -0,0 +1,50 @@
+package apply.application
+
+import apply.domain.applicant.Applicant
+import apply.domain.applicant.ApplicantRepository
+import apply.domain.applicant.Gender
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import javax.annotation.PostConstruct
+
+@Transactional
+@Service
+class ApplicantService(val applicantRepository: ApplicantRepository) {
+    fun findAll(): List<Applicant> {
+        return applicantRepository.findAll()
+    }
+
+    fun getByName(name: String): Applicant =
+        applicantRepository.findByName(name).orElse(null)
+
+    @PostConstruct
+    private fun populateDummy() {
+        if (applicantRepository.count() != 0L) {
+            return
+        }
+        val applicants = listOf(
+            Applicant(
+                "홍길동1",
+                "@email.com",
```

Comment:
더미 데이터긴 하지만 이메일 id가 빠져서 문제가 생길 수 도 있을 것 같아요 :)
```suggestion
                "a@email.com",
```

Reply:
나중에 유효성 검사를 위해 꼭 필요하겠네요. 👍